app/views/plots/index.html.erb

```diff
@@ -0,0 +1,12 @@
+<% @plots.each do |plot| %>
+    <section id="plot_<%= plot.id %>">
+    <br>
+    <p>Plot Number: <%= plot.number %>
+    <br>
+    <% plot.plants.each do |plant| %>
+        <br>
+        <%= plant.name %>
+        <%= button_to "Delete #{plant.name}", garden_plant_plot_path(plot.id, plant.id), method: :delete %>
+    <% end %>
+    </section>
```

Comment:
In this button, you're passing along the plot id and the plant id, and those id's will get stored in the params `garden_id` and `id` respectfully. So, in your controller, where you're calling `PlantPlot.find(params[:id]).destroy`, you are actually destroying a `PlantPlot` that has the id of the plant that you passed in. Due to the order in which you created these PlantPlot resources in your tests, this didn't pose an issue, but it's not doing what you want it to.